dna-frontend/src/app/core/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,96 @@
+<button
```

Comment:
- Add classes or ids to the elements to allow easier e2e testing

Reply:
made those changes